Ref.Windows.Tests/ViewModels/MainWindowViewModelTests.cs

```diff
@@ -307,6 +308,35 @@ public void CreateBook_CreatesAndEditsNewBook()
         }
 
         [Test]
+        public void CreateThesis_AsksIfEditing()
```

Comment:
These `AsksIfEditing` tests are so similar, they could probably be refactored to not repeat the common parts.